src/routes/index.tsx

```diff
@@ -74,19 +69,33 @@ const Routes = (): React.ReactElement => {
         path="/:url*(/+)"
         render={() => <Redirect to={location.pathname.replace(/\/+$/, `${location.search}${location.hash}`)} />}
       />
+
+      <Route
+        // Redirect /xdai root to /gno
+        path="/xdai"
+        render={() => <Redirect to="/gno" />}
+      />
+      <Route
+        // Redirect xdai: shortName to gno:
+        path="/xdai\::url*"
+        render={() => <Redirect to={location.pathname.replace(/\/xdai:/, `gno:`)} />}
+      />
+
       {
         // Redirection to open network specific welcome pages
-        getNetworkRootRoutes().map(({ chainId, route }) => (
+        getNetworkRootRoutes().map(({ chainId, route, shortName }) => (
           <Route
             key={chainId}
-            path={route}
+            // /xdai also sets chainId correctly
```

Comment:
The comment can be removed. All networks should behave the same, no need to mention xdai.